src/Microsoft.OData.Core/Microsoft.OData.Core.csproj

```diff
@@ -92,6 +92,7 @@
     <Compile Include="UriParser\SyntacticAst\ComputeToken.cs" />
     <Compile Include="UriParser\SyntacticAst\ComputeExpressionToken.cs" />
     <Compile Include="UriParser\TypeFacetsPromotionRules.cs" />
+    <Compile Include="Uri\ApplyClauseToStringBuilder.cs" />
     <Compile Include="ValidationKinds.cs" />

```

Comment:
This new file needs to be added to csprojs for other solutions. Please refer to similar change in this example for csprojs that need to be updated:
[https://github.com/OData/odata.net/commits/master/src/Microsoft.OData.Core/Build.NetStandard/Microsoft.OData.Core.NetStandard.csproj](https://github.com/OData/odata.net/commits/master/src/Microsoft.OData.Core/Build.NetStandard/Microsoft.OData.Core.NetStandard.csproj)